@bazel/webpack/webpack/worker/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+alias(
+    name = "webpack.config.js",
+    actual = "//webpack/worker:webpack.config.js",
```

Comment:
oh I see, I figured we'd just give up on the `/webpack` directory since we need to align with `@bazel/` in userland. but then we aren't following the style guide.

If we have indirection like this we should at least have some comments in the file so we remember why

Reply:
yes so let's add some comments explaining this to our future selves, like in `@bazel/webpack/README.md`

Like "this folder is just here for local development within our repo, so that paths intended for use after the package is installed are the same as what we see locally" or sth